packages/components/src/components/as-range-slider/slider/as-range-slider.tsx

```diff
@@ -302,6 +302,27 @@ export class RangeSlider {
   }
 
   private _getStepValue(value) {
-    return Math.round(value / this.step) * this.step;
+    const stepValue = (value / this.step) * this.step;
+    return this.roundToStep(stepValue, this.step);
   }
+
+  private roundToStep(numberToRound: number, step: number) {
+    return Number.parseFloat(numberToRound.toFixed(getDecimalPlaces(step)));
+  }
+}
+
+function getDecimalPlaces(decimalNumber): number {
```

Comment:
why outside the class?

Reply:
I thought It didn't belong to that specific class, but I should've put it in utilities! 😥